packages/expo-dev-launcher/bundle/queries/useBranchesForApp.tsx

```diff
@@ -154,9 +154,23 @@ export function useBranchesForApp(appId: string) {
 
   React.useEffect(() => {
     if (query.error) {
-      toastStack.push(() => (
-        <Toasts.Error>Something went wrong trying to fetch branches for this app</Toasts.Error>
-      ));
+      const doesNotHaveErrorShowing =
+        toastStack.getItems().filter((i) => i.status === 'pushing' || i.status === 'settled')
+          .length === 0;
+
+      // @ts-ignore
+      const errorMessage = query.error.message;
+
+      if (doesNotHaveErrorShowing) {
+        toastStack.push(() => (
+          <Toasts.Error>
+            <Text color="error" size="small">
+              {errorMessage?.slice(0, 80) ||
```

Comment:
Are you doing this slice to prevent a giant error from taking over the whole screen? If so, I'm wondering if 80 might be a little bit overly conservative. (as an example, it looks like the message in your screenshot is exactly 80 chars, seems a little too soon to be bumping up against a limit?)

Reply:
that's true - yes the whole message takes up the whole screen - maybe I just restrict the number of lines in this case, I feel like this was my original intention and kind of lost sight of it I guess